Cargo.lock

```diff
@@ -4361,6 +4426,15 @@ dependencies = [
  "serde_json",
 ]
 
+[[package]]
+name = "rust-base58"
+version = "0.0.4"
```

Comment:
Where dose this come from? Seems like a weird dependency.

Reply:
`near-jsonrpc-fuzz`: https://github.com/near/nearcore/blob/376791e6783aa72cfd93665770a9fa865a04df69/chain/jsonrpc/fuzz/fuzz_targets/fuzz_target_1.rs#L126

Reply:
We use `bs58` elsewhere, let's update the fuzz tests to use that as well?